File: chap07/fig7-5-left.R
library(ggplot2)

d <- read.csv('input/data-conc.csv')

p <- ggplot(data=d, aes(x=Time, y=Y)) +
  theme_bw(base_size=18) +
  geom_line(size=1) +
  geom_point(size=3) +
  labs(x='Time (hour)', y='Y') +
  scale_x_continuous(breaks=d$Time, limits=c(0,24)) +
  ylim(-2.5, 16)
ggsave(p, file='output/fig7-5-left.png', dpi=300, w=4, h=3)

File: chap07/fig7-5-left.py
import pandas
from plotnine import ggplot, aes, theme_bw, geom_line, geom_point, labs, scale_x_continuous, ylim

d = pandas.read_csv('input/data-conc.csv')

p = (ggplot(d, aes('Time', 'Y'))
    + theme_bw(base_size=18)
    + geom_line(size=1)
    + geom_point(size=3)
    + labs(x='Time (hour)', y='Y')
    + scale_x_continuous(breaks=d.Time, limits=[0,24])
    + ylim(-2.5, 16)
)
p.save(filename='output/fig7-5-left.py.png', dpi=300, width=4, height=3)
